server/apps/publish/publish_filters/filter_condition.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+import re
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+
+logger = logging.getLogger(__name__)
+
+
+class FilterConditionResource(Resource):
+    schema = {
+        'name': {
+            'type': 'string',
+            'nullable': False,
+        },
+        'field': {
+            'type': 'string',
+            'nullable': False,
+            'allowed': ['anpa-category',
+                        'urgency',
+                        'keywords',
+                        'priority',
+                        'slugline',
+                        'type',
+                        'source',
+                        'headline',
+                        'body_html',
+                        'genre'],
+        },
+        'operator': {
+            'type': 'string',
+            'allowed': ['in',
+                        'nin',
+                        'like',
+                        'notlike',
+                        'startswith',
+                        'endswith'],
+            'nullable': False,
+        },
+        'value': {
+            'type': 'string',
+            'nullable': False,
+        }
+    }
+
+    additional_lookup = {
+        'url': 'regex("[\w,.:-]+")',
+        'field': 'name'
+    }
+
+    datasource = {'default_sort': [('_created', -1)]}
+    privileges = {'POST': 'publish_queue', 'PATCH': 'publish_queue'}
+
+
+class FilterConditionService(BaseService):
+    def on_create(self, docs):
+        pass
```

Comment:
Is this needed? The `on_create` method already exists in `BaseService` (and already does nothing).

Or is the override intentional, so that indeed nothing is done in this method?